static/styles/app_components.css

```diff
@@ -407,7 +407,7 @@
 
 .stream_header_colorblock {
     @extend .stream-selection-header-colorblock;
-    margin-bottom: 5px;
+    margin-bottom: 20px;
```

Comment:
I'm pretty sure this is going to break any other widget using the `stream_header_colorblock` CSS class.

Reply:
Thanks, I should have checked this obvious change within other places as well.

Reply:
Fixed this in c343b1c98b6660eb3a06bcb1afd24c2741edb1f4.